src/containers/App/App.js

```diff
@@ -25,6 +26,7 @@ function App() {
           dispatch(setJobs(data.data))
```

Comment:
Do we need to set isLoaded to true before we do the apiCall on line 24?

Reply:
Good catch! It should be `true`, forgot to change it back.